react-search/my-app/src/SearchBar.tsx

```diff
@@ -0,0 +1,16 @@
+type Props = {
+  searchTerm: string;
+  handleSearchChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
You could hide the event by passing the text as the parameter instead of the event:
```
type Props = {
  handleSearchChange: (input: string) => void;
}

// below
return <input type="search" onChange={(e) => handleSearchChange(e.target.value)} />;
```

I think your code is cleaner and easier to work with if you extract the information out of the event in the child components. That's a good pattern to always use.